scale-ui/app/modules/jobs/partials/jobDetailTemplate.html

```diff
@@ -27,11 +27,20 @@ <h4 class="panel-title error-title"><strong>{{ vm.job.error.category }} Error: {
                         {{ vm.job.error.description }}
                     </div>
                 </div>
+                <div class="panel panel-info" ng-show="vm.job.is_superseded">
+                    <div class="panel-heading">
+                        <div class="panel-title">
+                            <strong>SUPERSEDED {{ vm.supersededChange }}</strong>
+                        </div>
+                    </div>
+                </div>
```

Comment:
The job part might be missing. A job can be superseded without a specific job having superseded it. So if there is no "superseded by" job, then we want just SUPERSEDED instead of SUPERSEDED BY. Agree with @wardcr suggestions.